.buildkite/pipeline.yml

```diff
@@ -1,8 +1,22 @@
-  - name: 'Run Test Suite (:kubernetes: 1.7.5)'
+  - name: 'Run Test Suite (:kubernetes: 1.8.2)'
     command: bin/ci
     agents:
-      queue: pipa-production-minikube-v1.7.5
-  - name: 'Run Test Suite (:kubernetes: 1.6.4)'
+      queue: minikube-v1.8.2
+    env:
+      LOGGING_LEVEL: 4
```

Comment:
Is this for buildkite or ultimately for k8s? If the latter, should we set it in `bin/ci` like we do a few other env vars?

Reply:
`LOGGING_LEVEL` refers to the Kubernetes components. The default (if omitted) is 4. I included it here explicitly as documentation to make it obvious how to get a more verbose log.

Moving it to `bin/ci` is too late as the env var has to be present during the initial k8s bootstrap before actually executing the command.